lib/gemstash/version.rb

```diff
@@ -2,5 +2,5 @@
 
 # :nodoc:
 module Gemstash
-  VERSION = "2.7.1"
+  VERSION = "2.8.0"
```

Comment:
Is this something needed to make the test pass?

Reply:
no but I was going to cut a new version once main is green